### PR TITLE
docs: search for a product tutorial

### PR DESCRIPTION
**Pull request by Roxie-32**
### What
- Using the OFF Tutorial : Search for a products. 
### Part of
- #7446

### CODE REVIEW

**Comment by Roxie-32** [comment]
@alexgarel @stephanegigandet the search tutorial PR is open now

**Comment by Roxie-32** [comment]
@alexgarel @stephanegigandet you can review this now, I have added the tutorial example to sort a search response. I am moving on to add it to the Open APi.

File: docs/tutorials/using-the-OFF-API-tutorial.md [hunk 1]
```diff
@@ -225,4 +225,131 @@ For more details, see the reference documentation for [Add or Edit A Product](ht
...
+            ],
+            "code": "4056489641018",
+            "nutrition_grades": "c"
+        }
+    ],
+    "skip": 0
+}
+```
+
+To see other examples of sorting a search response, see the reference documentation for [Search for Products](https://openfoodfacts.github.io/openfoodfacts-server/reference/api/#tag/Read-Requests/operation/get-searchl)
```
Comment (alexgarel) [inline docs/tutorials/using-the-OFF-API-tutorial.md:355]: Typo:
```suggestion
To see other examples of sorting a search response, see the reference documentation for [Search for Products](https://openfoodfacts.github.io/openfoodfacts-server/reference/api/#tag/Read-Requests/operation/get-search)
```

**Review by alexgarel — APPROVED** [review]
LGTM.

**Pull request merged**